### PR TITLE
By default, make build.cmd run all tests as to ensure nothing is missed.

### PR DESCRIPTION
**Pull request by robward-ms**
### Description
By default, make build.cmd run all tests as to ensure nothing is missed.
Optionally, run build.cmd quick to run only the unit tests.

### Checklist (Uncheck if it is not completed)
- [   ] Test cases added
- [ x ] Build and test with one-click build and test script passed

### Additional work necessary
None

### CODE REVIEW

File: build.cmd [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@echo off
```
Comment (robward-ms) [inline build.cmd]: The @echo off should still ne part of the file. Not ready to merge.

Comment (robward-ms) [inline build.cmd]: Fixed.

File: build.cmd [hunk 2]
```diff
@@ -12,21 +12,41 @@ REM is not supported. Also handle x86 operating systems, where %ProgramFiles(x86
 REM %MSBuild% value when setting the variable and never quote %MSBuild% references.
 set MSBuild="%ProgramFiles(x86)%\MSBuild\12.0\Bin\MSBuild.exe"
 if not exist %MSBuild% @set MSBuild="%ProgramFiles%\MSBuild\12.0\Bin\MSBuild.exe"
+set FullBuild=0
 
-if /I "%1" == "" goto BuildDefaults
+REM with no switches, run the full set of tests.
+if /I "%1" == "" (
```
Comment (biaol-odata) [inline build.cmd]: Can we combine this non-switches case with the "FULL" case below since they are equivalent?

Comment (robward-ms) [inline build.cmd]: Sadly, no. Cmd IF syntax only supports the NOT operator: https://ss64.com/nt/if.html

File: build.cmd [hunk 3]
```diff
@@ -12,21 +12,41 @@ REM is not supported. Also handle x86 operating systems, where %ProgramFiles(x86
...
+  set FullBuild=1
+  goto BuildDefaults
+)
+
+REM support quick build, unit tests only.
+if /I "%1" == "QUICK" (
```
Comment (robward-ms) [inline build.cmd]: Are you suggesting "quick" and "q" or just "q"? What about local customization, i.e. alias?

Comment (xuzhg) [inline build.cmd]: Yes. I think to add `-q` as an alternative to `QUICK` maybe easy to use. For someone who has local customization, he can use `QUICK`, else can use `-q` quickly. Any thoughts?

Comment (robward-ms) [inline build.cmd]: Fixed.

**Pull request merged**